### PR TITLE
Feat: pairing statistic ikp and aduan

### PR DESCRIPTION
**Pull request by rayfajars**
## FEAT
pairing statistik api IKP

## Preview
image

## Evidence
title: pairing statistic ikp
project: Jabar Super Apps
participants: @rayfajars @yoslie @agunghide @Ibwedagama @doohanas

### CODE REVIEW

File: components/Aduan/DaftarIKP/index.vue [hunk 1]
```diff
@@ -364,6 +358,26 @@ export default {
     },
     goToPageDetail (id) {
       this.$router.push(`/aduan/penginputan-ikp/detail-ikp/${id}`)
+    },
+    async getCount () {
+      const queryCount = { ...this.query }
+      queryCount.status = ''
```
Comment (Ibwedagama) [inline components/Aduan/DaftarIKP/index.vue:362]: just merge this code together

```suggestion
      const queryCount = { ...this.query, status: '' }
```

File: components/Aduan/DaftarIKP/index.vue [hunk 2]
```diff
@@ -289,7 +281,9 @@ export default {
       this.$refs.datepicker.closePopup()
     },
     setQuery (params) {
+      console.log(params)
       this.query = { ...this.query, ...params }
+      console.log(this.query, 'query ikp')
```
Comment (yoslie) [inline components/Aduan/DaftarIKP/index.vue:286]: unused console log

File: components/Aduan/index.vue [hunk 3]
```diff
@@ -315,6 +315,8 @@ export default {
   },
   async fetch () {
     try {
+      console.log(JSON.stringify(Object.keys(this.query)).match('complaint_status_id'), 'true')
+      console.log(!JSON.stringify(Object.keys(this.query)).match('complaint_status_id'))
```
Comment (yoslie) [inline components/Aduan/index.vue:319]: another console log

Comment (Ibwedagama) [inline components/Aduan/index.vue]: please remove this console.log

File: components/Aduan/index.vue [hunk 4]
```diff
@@ -564,29 +546,22 @@ export default {
     addComplaintStatusFilterHandle () {
       const listValueStatusComplaint = this.getStatusComplaintByComplaintType()
       for (let i = 0; i < listValueStatusComplaint.length; i++) {
+        // console.log(listValueStatusComplaint[i])
         this.setQuery({
           [`complaint_status_id[${i - 1}]`]: listValueStatusComplaint[i].id
         })
       }
+      // console.log(this.query)
```
Comment (yoslie) [inline components/Aduan/index.vue:554]: this one too

**Review by yoslie — CHANGES_REQUESTED** [review]

**Review by yoslie — APPROVED** [review]

File: components/Aduan/index.vue (hunk 4 above)
Comment (rayfajars) [reply components/Aduan/index.vue]: fix [here](https://github.com/jabardigitalservice/super-app-cms/pull/181/commits/5c9c9337e68b2dea187fb219e35f3a31bfa01bd8)

File: components/Aduan/index.vue (hunk 3 above)
Comment (rayfajars) [reply components/Aduan/index.vue]: fix [here](https://github.com/jabardigitalservice/super-app-cms/pull/181/commits/5c9c9337e68b2dea187fb219e35f3a31bfa01bd8)

File: components/Aduan/DaftarIKP/index.vue (hunk 2 above)
Comment (rayfajars) [reply components/Aduan/DaftarIKP/index.vue]: fix [here](https://github.com/jabardigitalservice/super-app-cms/pull/181/commits/5c9c9337e68b2dea187fb219e35f3a31bfa01bd8)

**Review by agunghide — APPROVED** [review]

**Review by doohanas — APPROVED** [review]

**Review by Ibwedagama — APPROVED** [review]

File: components/Aduan/DaftarIKP/index.vue (hunk 2 above)
[the remark this answers is not in the document]
Comment (rayfajars) [reply components/Aduan/DaftarIKP/index.vue]: sudah saya perbaiki di comment sebelumnya ya bli @Ibwedagama

File: components/Aduan/DaftarIKP/index.vue (hunk 1 above)
Comment (rayfajars) [reply components/Aduan/DaftarIKP/index.vue:362]: sudah saya perbaiki di comment sebelumnya ya bli @Ibwedagama

File: components/Aduan/index.vue (hunk 3 above)
Comment (rayfajars) [reply components/Aduan/index.vue]: sudah saya perbaiki di comment sebelumnya ya bli @Ibwedagama

**Pull request merged**